### DISCUSSION TITLE
Y'all are JIT. Just-Incredibly-Terrific.

### DISCUSSION THREAD

**Discussion by raybrownco** [tailwindlabs/tailwindcss#3801] [Kind words]
I've been kicking the tires on the [JIT version](https://blog.tailwindcss.com/just-in-time-the-next-generation-of-tailwind-css) since it's announcement last week, and I'd just like to say that I'm blown away. It's an incredible tool that has worked out of the box with zero issues so far, and I've noticed a tremendous improvement in my devtools experience thanks to it. Thank you so much for your hard work on such an awesome part of the Tailwind tool belt.

**Comment by adamwathan** [comment]
Thank you so much @raybrownco! 🙌
